CHANGELOG.md

```diff
@@ -25,6 +25,10 @@
   [Aaron McTavish](https://github.com/aamctustwo)
   [#202](https://github.com/realm/SwiftLint/issues/202)
 
+* Add LegacyConstantRule.
```

Comment:
missing 2 trailing spaces as described in https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes.